packages/fxa-auth-server/config/index.js

```diff
@@ -569,6 +569,66 @@ const conf = convict({
     env: 'DEVICE_NOTIFICATIONS_ENABLED',
     default: true
   },
+  subhub: {
+    enabled: {
+      doc: 'Indicates whether talking to the SubHub server is enabled',
+      format: Boolean,
+      default: false,
+      env: 'SUBHUB_ENABLED'
+    },
+    useStubs: {
+      doc: 'Indicates whether to use stub methods for SubHub instead of talking to the server',
+      format: Boolean,
+      default: false,
+      env: 'SUBHUB_USE_STUBS'
+    },
+    stubs: {
+      plans: {
+        doc: 'Stub data used for plans',
+        format: Array,
+        env: 'SUBHUB_STUB_PLANS',
+        default: []
+      }
+    },
+    url: {
+      doc: 'SubHub Server URL',
+      format: 'url',
+      default: 'https://subhub.services.mozilla.com/',
+      env: 'SUBHUB_URL'
+    },
+    key: {
+      doc: 'Authentication key to use when accessing SubHub server',
+      format: String,
+      default: 'YOU MUST CHANGE ME',
+      env: 'SUBHUB_KEY'
+    },
+  },
+  subscriptions: {
+    managementClientId: {
+      doc: 'OAuth client ID for management pages',
```

Comment:
Making the OAuth client ID for off-domain subscription management pages configurable. Kind of guessing it's necessary.

Reply:
So, I guess the thing I don't know here is: What's the process for creating a new OAuth client, and after doing so can I use that ID here for `dev.json` and remote tests? (Would it basically be the ["Becoming a Firefox Accounts relier"](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/Firefox_Accounts/Introduction#Becoming_a_Firefox_Accounts_relier) process from MDN?)

Reply:
Though, actually, if I'm getting rid of this route... the place where we'll need to deal with that client ID is in the client - i.e. the payment pages on the content server. That is, unless restricting access to the subscription management scope requires the client ID